src/language/index.tsx

```diff
@@ -0,0 +1,24 @@
+import en from './en.json';
+import sv from './sv.json';
+//"valuesHeaderEvaluation"
+//"activitiesButtonGradeDay"
+//"activitiesSliderGradeDay"
+//Format: "navigationbarTypeText"
+//navigationbar: activities, values, calendar, info
+
+//"january"
+//"february"
+//.
+//.
+//.
+//"monday"
```

Comment:
I would say remove this, or create a nicer comment about the format. I do think it might be necessary.